nixos/tests/stalwart-mail.nix

```diff
@@ -40,12 +40,12 @@ in import ./make-test-python.nix ({ lib, ... }: {
           };
         };
 
-        session.auth.mechanisms = [ "PLAIN" ];
-        session.auth.directory = "in-memory";
+        session.auth.mechanisms = [ [ "plain" ] ];
+        #session.auth.directory = "memory";
```

Comment:
I don't think that we need those lines commented out in the test.
I assume (I am just scrolling over atm) that those are the defaults in the module or by stalwart, thus we are no longer setting them here?